less/actors.less

```diff
@@ -501,10 +501,10 @@
     .item {
       .item-name {
         cursor: pointer;
-        &.rollable:hover .item-image {
+        &:is(.rollable, .usable):hover .item-image {
```

Comment:
Can we move to using `.usable` directly rather than needing to support both `.rollable` and `.usable`? OR can we leave CSS alone here and just keep using `.rollable` and make this just an JS API change for now?

Reply:
I'll just leave the CSS alone for now.